oop/src/views/MoveInfoPage.vue

```diff
@@ -210,6 +210,17 @@ export default {
       }, 2000);
     }
   },
+  watch: {
+    country(newValue) {
+      if (newValue) {
+        this.addressLine1 = null;
+        this.addressLine2 = null;
+        this.province = null;
+        this.city = null;
+        this.postalCode = null;
+      }
+    },
```

Comment:
There's a bug with this `watch` entry. When the user fills out the fields on the "Move Information" page, then continues to the "Review" page, then clicks "Edit" to go back to the "Move Information" page, the fields are now empty.

Check out how I did the `watch` entries on the `AccountTypePage` component. I added a check on a variable `isPageLoaded` and only if `true`, reset the fields.

https://github.com/bcgov/MOH-AOP-OOP/blob/main/oop/src/views/AccountTypePage.vue#L300-L303